include/class.task.php

```diff
@@ -172,16 +172,6 @@ function getCloseDate() {
         return $this->isClosed() ? $this->closed : '';
     }
 
-    function getStatusId() {
```

Comment:
I decided this wasn't a good way to do things, so I got rid of it altogether. Can't rely on the status names staying the same...